doc/integration/github.md

```diff
@@ -18,6 +18,7 @@ GitHub will generate an application ID and secret key for you to use.
     - Homepage URL: The URL to your GitLab installation. 'https://gitlab.company.com'
     - Application description: Fill this in if you wish.
     - Authorization callback URL: 'https://gitlab.company.com/'
+    - If install from source, default callback URL is '${YOUR_DOMAIN}/import/github/callback'
```

Comment:
Yes, it's the same in Omnibus as well. Remove `If install from source` and we can merge :)